bin/uwsgi.ini

```diff
@@ -0,0 +1,8 @@
+[uwsgi]
+module = basket.wsgi:application
+socket = :8001
```

Comment:
How come port 8001 here? Does anything else need adjusting from 8000 to 8001 to match?

Reply:
Good call out. Reading the docs it's actually better to use a unix socket than a port. I might need some SRE guidance as to whether we have nginx in front of this or a load balancer to get these to line up. Oddly this worked fine when I deployed it to the dev server for testing.